src/utils/validators.js

```diff
@@ -560,7 +562,7 @@ const validateContactType = (contactType, value) => {
       break
     }
     case "other": {
-      if (value.length > CONTACT_DESCRIPTION_MAX_LENGTH) {
+      if (getLengthWithoutTags(value) > CONTACT_DESCRIPTION_MAX_LENGTH) {
         errorMessage = `Description should be shorter than ${CONTACT_DESCRIPTION_MAX_LENGTH} characters.`
```

Comment:
I think we shouldn't be allowing iframe inside the contact us page - we can leave these as the original restriction

Reply:
makes sense - have made the change!